enterprise/app/filter/filter.tsx

```diff
@@ -147,10 +213,13 @@ export default class FilterComponent extends React.Component<FilterProps, State>
                     <Radio value="CI" checked={roleValue === "CI"} />
                     <span className="role-badge CI">CI</span>
                   </label>
-                  <label onClick={this.onRoleChange.bind(this, "CI_RUNNER")}>
-                    <Radio value="CI_RUNNER" checked={roleValue === "CI_RUNNER"} />
-                    <span className="role-badge CI_RUNNER">Workflow</span>
-                  </label>
+                  {/* TODO(bduffany): Figure out how to deal with workflows for the trends page */}
```

Comment:
Can we just leave this in instead of adding special case logic?

Reply:
Done -- after https://github.com/buildbuddy-io/buildbuddy/pull/861 is submitted the concern here will go away.